state/graphql-sender/src/main/java/com/expediagroup/streamplatform/streamregistry/state/graphql/GraphQLConverter.java

```diff
@@ -116,6 +119,11 @@ default SpecificationInput convertSpecification(S specification) {
               .collect(toList()))
           .type(specification.getType())
           .configuration(specification.getConfiguration())
+          .security(specification.getSecurity().entrySet().stream().map(
```

Comment:
Is there a complementary side to this that converts from the input list to a map?

Reply:
There is now! See the changes I made to the SpecificationInput class, it has the `asSpecification` method which takes the list of Roles and converts to the Map Role -> List<Principal>